### DISCUSSION TITLE
How can I reset column widths?

### DISCUSSION THREAD

**Discussion by akraines** [TanStack/table#2544] [General]
I'm using useResizeColumns plugin. I want to add a button to my table which will reset the column widths. I tried using controlled state to reset the state.columnResizing object to its initial state and it didn't update the column widths.
What is the correct approach?

Thanks

**Comment by gargroh** [comment]
PR for the same exists - https://github.com/tannerlinsley/react-table/pull/2461

**Comment by akraines** [comment]
Perfect! Thanks. How can I use this in the meantime? Manually add the updated version of the file to my local project and use the local version of the plugin?

**Comment by gargroh** [comment]
once chk with Tanner before forking...

**Comment by tannerlinsley** [comment]
Merged.
